scripts/client_test.sh

```diff
@@ -4,6 +4,10 @@ JOB_TYPE=$1
 PS_NUM=$2
 WORKER_NUM=$3
 
+# Generate checkpoint for mnist to test evaluation and prediction
+MNIST_CKPT_DIR=${PWD}/mnist/
+python gen_mnist_checkpoint.py --checkpoint_dir=${MNIST_CKPT_DIR}
```

Comment:
I didn't see this python file `gen_mnist_checkpoint.py` in the PR

Reply:
Oh, sorry. The file is omitted and I have submitted it.